google/cloud/pubsublite/internal/resumable_async_streaming_read_write_rpc.h

```diff
@@ -59,6 +59,34 @@ using RetryPolicyFactory = std::function<std::unique_ptr<RetryPolicy>()>;
 /**
  * The purpose of this class is to encapsulate the retry logic of a generic
  * bidirectional asynchronous RPC stream.
+ *
+ * Example (unrealistic) usage:
+ * ```
+ * std::string ExampleReadUsage() {
+ *   ResumableAsyncStreamingReadWriteRpc<Request, Response> resumable_stream;
+ *   auto start = resumable_stream.Start();
+ *   size_t num_attempts = 0;
+ *   while (num_attempts < 3) {
```

Comment:
this shouldn't be a loop.

Reply:
https://github.com/googleapis/google-cloud-cpp/pull/8441#discussion_r814015630